pkg/cloudprovider/providers/azure/azure_util_vmss.go

```diff
@@ -76,15 +76,17 @@ type scaleSet struct {
 	cacheMutex sync.Mutex
 	// A local cache of scale sets. The key is scale set name and the value is a
 	// list of virtual machines belonging to the scale set.
-	cache map[string][]scaleSetVMInfo
+	cache                     map[string][]scaleSetVMInfo
```

Comment:
nit: too many spaces here

Reply:
I think gofmt indents it with next field `availabilitySetNodesCache `